pkg/asset/agent/manifests/agentclusterinstall_test.go

```diff
@@ -129,6 +129,14 @@ func TestAgentClusterInstall_Generate(t *testing.T) {
 		installConfigOverrides: `{"platform":{"baremetal":{"hosts":[{"name":"control-0.example.org","bmc":{"username":"bmc-user","password":"password","address":"172.22.0.10","disableCertificateVerification":true},"role":"master","bootMACAddress":"98:af:65:a5:8d:01","hardwareProfile":""},{"name":"control-1.example.org","bmc":{"username":"user2","password":"foo","address":"172.22.0.11","disableCertificateVerification":false},"role":"master","bootMACAddress":"98:af:65:a5:8d:02","hardwareProfile":""},{"name":"control-2.example.org","bmc":{"username":"admin","password":"bar","address":"172.22.0.12","disableCertificateVerification":true},"role":"master","bootMACAddress":"98:af:65:a5:8d:03","hardwareProfile":""}],"clusterProvisioningIP":"172.22.0.3","provisioningNetwork":"Managed","provisioningNetworkInterface":"eth0","provisioningNetworkCIDR":"172.22.0.0/24","provisioningDHCPRange":"172.22.0.10,172.22.0.254"}}}`,
 	})
 
+	installConfigWithTrustBundlePolicy := getValidOptionalInstallConfig()
+	installConfigWithTrustBundlePolicy.Config.AdditionalTrustBundlePolicy = types.PolicyAlways
+
+	goodTrustBundlePolicyACI := getGoodACI()
+	goodTrustBundlePolicyACI.SetAnnotations(map[string]string{
+		installConfigOverrides: `{"additionalTrustBundlePolicy":"Always"}`,
```

Comment:
I guess the it could be updated as well for that

Reply:
Per our discussion, will add this as part of an additional PR to update the integration tests to include disconnected configurations, see https://issues.redhat.com/browse/OCPBUGS-44066